src/frontend/src/routes/(app)/post/[id]/+page.svelte

```diff
@@ -30,3 +35,90 @@
     {/each}
   {/if}
 {/await}
+<div>
+  <form class="modal-form m-5">
+    <label class="label">
+      <textarea
+        class="textarea px-3 py-2"
+        rows="3"
+        bind:value={reply}
+        placeholder="Start typing your reply"
+      />
+    </label>
+    <div class="flex justify-end">
+      <button class="btn variant-filled mt-2">Post reply</button>
+    </div>
+  </form>
+</div>
+<div class="m-5">
+  <!-- Only display comment to the post/event displayed, to see comments on the comment user needs to click on the comment to go down the rabbit hole -->
+  <div class="grid grid-cols-[auto_1fr] gap-2 my-4">
+    <!-- Link to the author -->
+    <a href="#">
```

Comment:
```suggestion
    <a href="#TODO">
```
+ other places, it breaks stuff then

Reply:
this was also fixed by latest commit